modules/tax/tests/src/TaxBrowserTestBase.php

```diff
@@ -1,13 +1,13 @@
 <?php
 
-namespace Drupal\commerce_tax\Tests;
+namespace Drupal\Tests\commerce_tax;
```

Comment:
The new location should be tests/src/Functional, namespace Drupal\Tests\commerce_tax\Functional, since we're extending the functional test base.